packages/back-end/generated/spec.yaml

```diff
@@ -604,19 +607,24 @@ paths:
                 properties:
                   result:
                     $ref: '#/components/schemas/ExperimentResults'
-  /visual-changesets:
+  '/experiments/{id}/visual-changesets':
     get:
-      summary: Get all visual changeset
+      summary: Get all visual changesets
       tags:
         - visual-changesets
       parameters:
-        - $ref: '#/components/parameters/limit'
-        - $ref: '#/components/parameters/offset'
+        - id: null
+          name: id
+          in: path
+          required: true
+          description: The experiment id the visual changesets belong to
+          schema:
+            type: string
       operationId: listVisualChangesets
       x-codeSamples:
         - lang: cURL
           source: |
-            curl https://api.growthbook.io/api/v1/visual-changesets \
+            curl https://api.growthbook.io/api/v1/experiments/exp_123abc/visual-changesets \
```

Comment:
for the experiment ID, is there a better way to show that it's variable in the examples, instead of hardcoding, in case it adds confusion?

Reply:
I think the hardcoded id is fine in this case ... Because we do display the param variable prominently above the code sample. This is how it's being done with existent endpoints ... Take a look at experiment results for example:

<img width="499" alt="image" src="https://user-images.githubusercontent.com/2374625/223198876-78e6206e-cc27-47af-8578-0a77e71048e3.png">